packages/create-block/README.md

```diff
@@ -1,18 +1,18 @@
 # Create Block
 
-Create Block is an officially supported way to create blocks for registering a block for a WordPress plugin. It offers a modern build setup with no configuration. It generates PHP, JS, CSS code, and everything else you need to start the project.
+Create Block is an officially supported tool for scaffolding WordPress plugins with blocks. It generates PHP, JS, CSS code, and everything you need to start the project. It integrates a modern build setup with no configuration.
 
 It is largely inspired by [create-react-app](https://create-react-app.dev/docs/getting-started). Major kudos to [@gaearon](https://github.com/gaearon), the whole Facebook team, and the React community.
 
 ## Description
 
-Blocks are the fundamental element of the WordPress block editor. They are the primary way in which plugins and themes can register their own functionality and extend the capabilities of the editor.
+Blocks are the fundamental element of the WordPress block editor. They are the primary way in which plugins and themes can register their functionality and extend the editor's capabilities.
```

Comment:
Being able to register blocks from a theme isn't technically correct. The enqueuing process of registering a block assumes that the base path is from a plugin ( [see here](https://github.com/WordPress/wordpress-develop/blob/67d7e5040786bd8eb69410afe867c016cda3b10d/src/wp-includes/blocks.php#L112) ). It is possible to do it, but not without filtering the plugin URL

Reply:
Yes, let's remove "themes" from this sentence 👍🏻

Reply:
Done in https://github.com/WordPress/gutenberg/pull/39049/commits/57eb4fe7e448a5bcb9c977149b9b106204904c8c.